src/modules/settings/components/settings/global/Emotes.jsx

```diff
@@ -85,7 +85,7 @@ function EmotesModule() {
               {formatMessage(
                 {
                   defaultMessage:
-                    'Emote modifiers allow you to transform emotes in realtime. Wide: <code>w! emoteName</code>, Horizontal Flip: <code>h! emoteName</code>, Vertical Flip: <code>v! emoteName</code>, Zero-Width: <code>z! emoteName</code>',
+                    'Emote modifiers allow you to transform emotes in realtime. Wide: <code>w! emoteName</code>, Horizontal Flip: <code>h! emoteName</code>, Vertical Flip: <code>v! emoteName</code>, Zero-Width: <code>z! emoteName</code>, Rotate to Left: <code>l! emoteName</code>, Rotate to Right: <code>r! emoteName</code>',
```

Comment:
Other changes seem good, think we could drop the "to" and just say Rotate Left and Rotate Right

Reply:
done